packages/teams-js/src/public/calendar.ts

```diff
@@ -54,15 +65,24 @@ export namespace calendar {
     return ensureInitialized(runtime) && runtime.supports.calendar ? true : false;
   }
 
+  /** Open calendar item parameters. */
   export interface OpenCalendarItemParams {
+    /** A string representing the unique ID of the calendar item to be opened. */
```

Comment:
>A string representing the unique ID of the calendar item to be opened. [](http://example.com/codeflow?start=8&length=70)

What format should this ID be in?

Reply:
Updated in latest commit

Reply:
I don't know anything about this function so I actually don't know whether the answer to this question is obvious: where would a developer get this ID from? Is it returned from some kind of "get" call? Or does it come from the graph?